internal/notify/kafka.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+package notify
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/moov-io/achgateway/internal/kafka"
+	"github.com/moov-io/achgateway/internal/service"
+	"github.com/moov-io/achgateway/pkg/compliance"
+	"github.com/moov-io/achgateway/pkg/models"
+	"github.com/moov-io/base/log"
+
+	"gocloud.dev/pubsub"
+)
+
+type Kafka struct {
+	publisher *pubsub.Topic
+	cfg       *service.KafkaConfig
+}
+
+func NewKafka(logger log.Logger, cfg *service.KafkaConfig) (*Kafka, error) {
```

Comment:
This needs to be in multi.go as well. Was the intention to publish events on a new topic?

Reply:
We would provision a given topic where we would publish the notification events. So we do need a new topic.